From 638118c3820d920a5649342a9a4bfbb2739e6423 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 27 Apr 2025 16:15:43 +0200
Subject: [PATCH] Revert "init.d/user: quit fast if it's failing rapidly"

Temporary workaround for https://github.com/OpenRC/openrc/issues/852

This reverts commit 6411b4b0cce1781da20ec0a6c0babc4bd30d1f3f.

---
 init.d/user.in | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/init.d/user.in b/init.d/user.in
index 5b52ce3d..3fe9e8a2 100644
--- a/init.d/user.in
+++ b/init.d/user.in
@@ -17,11 +17,6 @@ command="@RC_LIBEXECDIR@/bin/openrc-user"
 command_args="$user"
 notify="fd:3"
 
-# If it's rapidly failing, usually due to XDG_RUNTIME_DIR being unset,
-# we should just quit fast.
-respawn_max=3
-respawn_period=1
-
 start_pre() {
 	if [ "$user" = "$RC_SVCNAME" ]; then
 		eerror "${RC_SVCNAME} cannot be started directly. You must create"
